client/src/components/Button/Button.js

```diff
@@ -1,24 +1,18 @@
 import React from 'react';
 import { Button as BaseButton } from 'reactstrap';
-import PropTypes from 'prop-types';
 import classnames from 'classnames';
 import IconHOC from './IconHOC';
 
-const Button = ({ className, noText, children, ...props }) =>
+const Button = ({
+  className,
+  noText = false,
+  children,
+  ...props
+}) =>
   (<BaseButton
     className={classnames(className, { 'btn--no-text': noText })}
     aria-label={noText ? children : undefined}
     {...props}
   >{noText ? undefined : children}</BaseButton>);
 
-Button.propTypes = {
-  ...BaseButton.propTypes,
-  noText: PropTypes.bool
-};
```

Comment:
Why are the `propTypes` being removed? That seems unrelated?

Reply:
Opps, reverted